share/revng/test/configuration/revng/for-propagated-prototypes.yml

```diff
@@ -0,0 +1,15 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+commands:
+  - type: revng.test-propagated-prototypes
+    from:
+      - type: revng-qa.compiled-with-debug-info
+        filter: for-propagated-prototypes
+    command: |-
+      RESUME=$$(temp -d);
+      revng analyze --model-type-id-seed=0 --resume "$$RESUME" ImportBinary "$INPUT" -o /dev/null;
+      revng analyze --model-type-id-seed=0 --resume "$$RESUME" AddPrimitiveTypes "$INPUT" -o /dev/null;
+      revng analyze --model-type-id-seed=0 --resume "$$RESUME" DetectABI "$INPUT" |
```

Comment:
Thanks to `revng model compare`, tests should pass even without `--model-type-id-seed=0`.

Drop it.

Reply:
Updates?